parser/fsimporter.go

```diff
@@ -199,7 +199,7 @@ func (fs *FSImporter) Run(indexedFiles []*fpt.IndexedFile) {
 		fs.buildBeacons(uconnMap)
 
 		// build or update the FQDN Beacons Table
-		fs.buildFQDNBeacons(hostnameMap)
+		fs.buildFQDNBeacons(hostMap)
```

Comment:
FQDNBeacons now takes in the list of hosts in the current parse set rather than the list of DNS queries.